R/reportPreview.R

```diff
@@ -178,21 +178,23 @@ reportPreviewServer <- function(id, selected_pkg, maint_metrics, com_metrics,
         h5('Published:'), selected_pkg$published()
       )
     })
-    
+
     # Display the decision status of the selected package.
     output$decision_display <- renderUI({
-      
+      req(selected_pkg$name())
+
       tagList(
         h5('Overall risk:'),
-        ifelse(selected_pkg$decision() == '', 
+        ifelse(selected_pkg$decision() == '',
                'Pending',
-               selected_pkg$decision()))
+               selected_pkg$decision()),
+        h5('Score:'), selected_pkg$score())
```

Comment:
In the Sidebar we have the score labeled as "Risk". I personally think the label should be "Risk Score:" over just "Score:" but am open to differing opinions.

Reply:
@Jeff-Thompson12 I agree, the label Risk Score is better than just Score. I'll change this.

Reply:
Changed. (Commit ID - f0efc77)